rosetta-runtime/src/main/java/com/rosetta/model/lib/meta/RosettaMetaData.java

```diff
@@ -16,12 +15,6 @@ public interface RosettaMetaData<T extends RosettaModelObject> {
 
 	List<Validator<? super T>> dataRules(ValidatorFactory factory);
 	
-	// @Compat. This will be empty, as choice rules are now all data rules.
```

Comment:
Won't removing this cause backward compat issues with generated code?

Reply:
This has been out of use (not being generated) from even before blueprints, so no old generated code should contain it anymore.